reports/example.md

```diff
@@ -8,36 +8,41 @@ organisation: coala
 organisation_link : https://coala.io
 project: <project title>
 project_link: <project link>
+report_year: <year of completing GSoC>
 tarball: <tarball of patches' link>
+tarball_hash: <shasum of tarball>
 mentors: >
- [mentor1]<link1> & [mentor2]<link2>
+  [mentor1]<link1> & [mentor2]<link2>
 phase:
- - Bonding : <bonding milstone link>
- - Phase 1 : <phase1 milsetone link>
- - Phase 2 : <phase2 milsetone link>
- - Phase 3 : <phase3 milsetone link>
+  - Bonding : <bonding milstone link>
+  - Phase 1 : <phase1 milsetone link>
+  - Phase 2 : <phase2 milsetone link>
+  - Phase 3 : <phase3 milsetone link>
 bio: >
- <your bio goes here>
+  <your bio goes here>
 social:
- - GitHub:
-   - username: <github username>
-   - link: https://github.com/<github username>
- - GitLab:
-   - username: <gitlab username>
-   - link: https://gitlab.com/<gitlab username>
- - Gitter:
-   - username: <gitter username>
-   - link: https://gitter.im/<gitter username>
- - HackerRank:
-   - username: <hackerrank username>
-   - link: https://hackerrank.com/<hackerrank username>
+  - GitHub:
+    - username: <github username>
+    - link: https://github.com/<github username>
+  - gitlab:
+    - username: <gitlab username>
+    - link: https://gitlab.com/<gitlab username>
+  - gitter:
+    - username: <gitter username>
+    - link: https://gitter.im/<gitter username>
+  - hackerrank:
+    - username: <hackerrank username>
+    - link: https://hackerrank.com/<hackerrank username>
+  - openhub:
+    - username: <openhub username>
+    - link: https://openhub.net/accounts/<openhub username>
 email: <email>
 blog: <link to your blog>
 activity:
- - <number>:
-   - repo: <repo name>
-   - link: <link>
-   - details: >
+  - <number>:
```

Comment:
do a few example entries.

and the link should include a hash which is only 8 long.

`https://github.com/coala/cEPs/commit/e9f75878`

not

`https://github.com/coala/cEPs/commit/e9f7587856e1c236d37e96f54a0cb4d72069776d`

to avoid line length errors

Reply:
Nice idea! Done!